config/routes.rb

```diff
@@ -88,7 +88,7 @@
       end
 
       # this block is only required if subscription via chargebee is enabled
-      if ENV['BILLING_ENABLED']
+      if ENV['BILLING_ENABLED'] || Rails.env.development? || Rails.env.test?
```

Comment:
I would leave this check intact, so even while developing the behaviour is consistent. 
Let's try setting the environment variable inside the test  using something like the `climate_control` gem 
https://thoughtbot.com/blog/testing-and-environment-variables

Reply:
The climate_control gem didn't work here because the routes only get loaded once on start of the whole rspec test. I fixed the issue by setting the env variable now in spec_helper.rb